src/hooks.rs

```diff
@@ -21,26 +24,24 @@ pub(crate) fn run_hook(
     }
     debug!("Rendering script {:?} -> {:?}", location, script_file);
 
+    let target: TemplateTarget = std::env::temp_dir().join("dotter_temp").into();
     crate::actions::perform_template_deploy(
         location,
         &script_file,
-        &std::env::temp_dir().join("dotter_temp").into(),
+        &target,
         &mut crate::filesystem::RealFilesystem::new(false),
         handlebars,
         variables,
     )
     .context("deploy script")?;
 
-    debug!("Running script file ");
+    debug!("Running script file");
     let mut child = if cfg!(windows) {
-        Command::new(script_file)
+        Command::new(target.target)
```

Comment:
I don't see the whole code but previously there was a `with_extension("bat")` in the windows case and it seems to have been lost

Reply:
Yes when moving from `script_file` inside _cache_ to _real_ target I've missed reporting that extension. I'll append it